### PR TITLE
EOS-25493: Changed the agent for Alex

### PR DESCRIPTION
**Pull request by venkuppu-chn**
Signed-off-by: Venkatesh K <venkatesh.k@seagate.com>

# Problem Statement
Pre-built docker images is having conflict in the node packages with Alex packages.

# Design
Created the new agent in jenkins

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [x] Interface change (if any) are documented
- [x] Side effects on other features (deployment/upgrade)
- [x] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: jenkins/automation/alex.groovy [hunk 1]
```diff
@@ -1,7 +1,7 @@
 pipeline {
     agent {
           node {
-             label "docker-${os_version}-node"
+             label "ssc-vm-g4-rhev4-0465.colo.seagate.com"
```
Comment (mukul-seagate11) [inline jenkins/automation/alex.groovy:4]: dont hardcode the hostname as it will be a security risk

**Review by mukul-seagate11 — CHANGES_REQUESTED** [review]

File: jenkins/automation/alex.groovy [hunk 2]
```diff
@@ -33,10 +33,14 @@ pipeline {
...
-                  yum install epel-release
-                  yum install nodejs
-                  npm install alex --global
+                  yum install epel-release -y
+                  curl -sL https://rpm.nodesource.com/setup_12.x | sudo bash -
+                  yum install nodejs -y
+                  npm install alex --global -y
+                  virtualenv alex --python=python2.7
+                  source ./alex/bin/activate
                   pip install GitPython==2.1.15
```
Comment (shailesh-vaidya) [inline jenkins/automation/alex.groovy:42]: use `pip3`

Comment (venkuppu-chn) [reply jenkins/automation/alex.groovy:42]: Unfortunately, the script is written and testes in python2.7.

File: jenkins/automation/alex.groovy (hunk 1 above)
Comment (venkuppu-chn) [reply jenkins/automation/alex.groovy]: @mukul-seagate11 , Thanks for pointing out this. I have changed as a jenkins parameter now.

**Review by mukul-seagate11 — APPROVED** [review]
looks good

File: jenkins/automation/alex.groovy (hunk 2 above)
Comment (shailesh-vaidya) [reply jenkins/automation/alex.groovy:41]: Ok

**Review by shailesh-vaidya — APPROVED** [review]

**Review by gauravchaudhari02 — APPROVED** [review]

**Pull request merged**